<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />
<meta name="viewport" content="width=device-width, initial-scale=1">

<style type="text/css">
@font-face {
font-family: octicons-link;
src: url(data:font/woff;charset=utf-8;base64,d09GRgABAAAAAAZwABAAAAAACFQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABEU0lHAAAGaAAAAAgAAAAIAAAAAUdTVUIAAAZcAAAACgAAAAoAAQAAT1MvMgAAAyQAAABJAAAAYFYEU3RjbWFwAAADcAAAAEUAAACAAJThvmN2dCAAAATkAAAABAAAAAQAAAAAZnBnbQAAA7gAAACyAAABCUM+8IhnYXNwAAAGTAAAABAAAAAQABoAI2dseWYAAAFsAAABPAAAAZwcEq9taGVhZAAAAsgAAAA0AAAANgh4a91oaGVhAAADCAAAABoAAAAkCA8DRGhtdHgAAAL8AAAADAAAAAwGAACfbG9jYQAAAsAAAAAIAAAACABiATBtYXhwAAACqAAAABgAAAAgAA8ASm5hbWUAAAToAAABQgAAAlXu73sOcG9zdAAABiwAAAAeAAAAME3QpOBwcmVwAAAEbAAAAHYAAAB/aFGpk3jaTY6xa8JAGMW/O62BDi0tJLYQincXEypYIiGJjSgHniQ6umTsUEyLm5BV6NDBP8Tpts6F0v+k/0an2i+itHDw3v2+9+DBKTzsJNnWJNTgHEy4BgG3EMI9DCEDOGEXzDADU5hBKMIgNPZqoD3SilVaXZCER3/I7AtxEJLtzzuZfI+VVkprxTlXShWKb3TBecG11rwoNlmmn1P2WYcJczl32etSpKnziC7lQyWe1smVPy/Lt7Kc+0vWY/gAgIIEqAN9we0pwKXreiMasxvabDQMM4riO+qxM2ogwDGOZTXxwxDiycQIcoYFBLj5K3EIaSctAq2kTYiw+ymhce7vwM9jSqO8JyVd5RH9gyTt2+J/yUmYlIR0s04n6+7Vm1ozezUeLEaUjhaDSuXHwVRgvLJn1tQ7xiuVv/ocTRF42mNgZGBgYGbwZOBiAAFGJBIMAAizAFoAAABiAGIAznjaY2BkYGAA4in8zwXi+W2+MjCzMIDApSwvXzC97Z4Ig8N/BxYGZgcgl52BCSQKAA3jCV8CAABfAAAAAAQAAEB42mNgZGBg4f3vACQZQABIMjKgAmYAKEgBXgAAeNpjYGY6wTiBgZWBg2kmUxoDA4MPhGZMYzBi1AHygVLYQUCaawqDA4PChxhmh/8ODDEsvAwHgMKMIDnGL0x7gJQCAwMAJd4MFwAAAHjaY2BgYGaA4DAGRgYQkAHyGMF8NgYrIM3JIAGVYYDT+AEjAwuDFpBmA9KMDEwMCh9i/v8H8sH0/4dQc1iAmAkALaUKLgAAAHjaTY9LDsIgEIbtgqHUPpDi3gPoBVyRTmTddOmqTXThEXqrob2gQ1FjwpDvfwCBdmdXC5AVKFu3e5MfNFJ29KTQT48Ob9/lqYwOGZxeUelN2U2R6+cArgtCJpauW7UQBqnFkUsjAY/kOU1cP+DAgvxwn1chZDwUbd6CFimGXwzwF6tPbFIcjEl+vvmM/byA48e6tWrKArm4ZJlCbdsrxksL1AwWn/yBSJKpYbq8AXaaTb8AAHja28jAwOC00ZrBeQNDQOWO//sdBBgYGRiYWYAEELEwMTE4uzo5Zzo5b2BxdnFOcALxNjA6b2ByTswC8jYwg0VlNuoCTWAMqNzMzsoK1rEhNqByEyerg5PMJlYuVueETKcd/89uBpnpvIEVomeHLoMsAAe1Id4AAAAAAAB42oWQT07CQBTGv0JBhagk7HQzKxca2sJCE1hDt4QF+9JOS0nbaaYDCQfwCJ7Au3AHj+LO13FMmm6cl7785vven0kBjHCBhfpYuNa5Ph1c0e2Xu3jEvWG7UdPDLZ4N92nOm+EBXuAbHmIMSRMs+4aUEd4Nd3CHD8NdvOLTsA2GL8M9PODbcL+hD7C1xoaHeLJSEao0FEW14ckxC+TU8TxvsY6X0eLPmRhry2WVioLpkrbp84LLQPGI7c6sOiUzpWIWS5GzlSgUzzLBSikOPFTOXqly7rqx0Z1Q5BAIoZBSFihQYQOOBEdkCOgXTOHA07HAGjGWiIjaPZNW13/+lm6S9FT7rLHFJ6fQbkATOG1j2OFMucKJJsxIVfQORl+9Jyda6Sl1dUYhSCm1dyClfoeDve4qMYdLEbfqHf3O/AdDumsjAAB42mNgYoAAZQYjBmyAGYQZmdhL8zLdDEydARfoAqIAAAABAAMABwAKABMAB///AA8AAQAAAAAAAAAAAAAAAAABAAAAAA==) format('woff');
}
body {
-webkit-text-size-adjust: 100%;
text-size-adjust: 100%;
color: #333;
font-family: "Helvetica Neue", Helvetica, "Segoe UI", Arial, freesans, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
font-size: 16px;
line-height: 1.6;
word-wrap: break-word;
}
a {
background-color: transparent;
}
a:active,
a:hover {
outline: 0;
}
strong {
font-weight: bold;
}
h1 {
font-size: 2em;
margin: 0.67em 0;
}
img {
border: 0;
}
hr {
box-sizing: content-box;
height: 0;
}
pre {
overflow: auto;
}
code,
kbd,
pre {
font-family: monospace, monospace;
font-size: 1em;
}
input {
color: inherit;
font: inherit;
margin: 0;
}
html input[disabled] {
cursor: default;
}
input {
line-height: normal;
}
input[type="checkbox"] {
box-sizing: border-box;
padding: 0;
}
table {
border-collapse: collapse;
border-spacing: 0;
}
td,
th {
padding: 0;
}
* {
box-sizing: border-box;
}
input {
font: 13px / 1.4 Helvetica, arial, nimbussansl, liberationsans, freesans, clean, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
}
a {
color: #4078c0;
text-decoration: none;
}
a:hover,
a:active {
text-decoration: underline;
}
hr {
height: 0;
margin: 15px 0;
overflow: hidden;
background: transparent;
border: 0;
border-bottom: 1px solid #ddd;
}
hr:before {
display: table;
content: "";
}
hr:after {
display: table;
clear: both;
content: "";
}
h1,
h2,
h3,
h4,
h5,
h6 {
margin-top: 15px;
margin-bottom: 15px;
line-height: 1.1;
}
h1 {
font-size: 30px;
}
h2 {
font-size: 21px;
}
h3 {
font-size: 16px;
}
h4 {
font-size: 14px;
}
h5 {
font-size: 12px;
}
h6 {
font-size: 11px;
}
blockquote {
margin: 0;
}
ul,
ol {
padding: 0;
margin-top: 0;
margin-bottom: 0;
}
ol ol,
ul ol {
list-style-type: lower-roman;
}
ul ul ol,
ul ol ol,
ol ul ol,
ol ol ol {
list-style-type: lower-alpha;
}
dd {
margin-left: 0;
}
code {
font-family: Consolas, "Liberation Mono", Menlo, Courier, monospace;
font-size: 12px;
}
pre {
margin-top: 0;
margin-bottom: 0;
font: 12px Consolas, "Liberation Mono", Menlo, Courier, monospace;
}
.select::-ms-expand {
opacity: 0;
}
.octicon {
font: normal normal normal 16px/1 octicons-link;
display: inline-block;
text-decoration: none;
text-rendering: auto;
-webkit-font-smoothing: antialiased;
-moz-osx-font-smoothing: grayscale;
-webkit-user-select: none;
-moz-user-select: none;
-ms-user-select: none;
user-select: none;
}
.octicon-link:before {
content: '\f05c';
}
.markdown-body:before {
display: table;
content: "";
}
.markdown-body:after {
display: table;
clear: both;
content: "";
}
.markdown-body>*:first-child {
margin-top: 0 !important;
}
.markdown-body>*:last-child {
margin-bottom: 0 !important;
}
a:not([href]) {
color: inherit;
text-decoration: none;
}
.anchor {
display: inline-block;
padding-right: 2px;
margin-left: -18px;
}
.anchor:focus {
outline: none;
}
h1,
h2,
h3,
h4,
h5,
h6 {
margin-top: 1em;
margin-bottom: 16px;
font-weight: bold;
line-height: 1.4;
}
h1 .octicon-link,
h2 .octicon-link,
h3 .octicon-link,
h4 .octicon-link,
h5 .octicon-link,
h6 .octicon-link {
color: #000;
vertical-align: middle;
visibility: hidden;
}
h1:hover .anchor,
h2:hover .anchor,
h3:hover .anchor,
h4:hover .anchor,
h5:hover .anchor,
h6:hover .anchor {
text-decoration: none;
}
h1:hover .anchor .octicon-link,
h2:hover .anchor .octicon-link,
h3:hover .anchor .octicon-link,
h4:hover .anchor .octicon-link,
h5:hover .anchor .octicon-link,
h6:hover .anchor .octicon-link {
visibility: visible;
}
h1 {
padding-bottom: 0.3em;
font-size: 2.25em;
line-height: 1.2;
border-bottom: 1px solid #eee;
}
h1 .anchor {
line-height: 1;
}
h2 {
padding-bottom: 0.3em;
font-size: 1.75em;
line-height: 1.225;
border-bottom: 1px solid #eee;
}
h2 .anchor {
line-height: 1;
}
h3 {
font-size: 1.5em;
line-height: 1.43;
}
h3 .anchor {
line-height: 1.2;
}
h4 {
font-size: 1.25em;
}
h4 .anchor {
line-height: 1.2;
}
h5 {
font-size: 1em;
}
h5 .anchor {
line-height: 1.1;
}
h6 {
font-size: 1em;
color: #777;
}
h6 .anchor {
line-height: 1.1;
}
p,
blockquote,
ul,
ol,
dl,
table,
pre {
margin-top: 0;
margin-bottom: 16px;
}
hr {
height: 4px;
padding: 0;
margin: 16px 0;
background-color: #e7e7e7;
border: 0 none;
}
ul,
ol {
padding-left: 2em;
}
ul ul,
ul ol,
ol ol,
ol ul {
margin-top: 0;
margin-bottom: 0;
}
li>p {
margin-top: 16px;
}
dl {
padding: 0;
}
dl dt {
padding: 0;
margin-top: 16px;
font-size: 1em;
font-style: italic;
font-weight: bold;
}
dl dd {
padding: 0 16px;
margin-bottom: 16px;
}
blockquote {
padding: 0 15px;
color: #777;
border-left: 4px solid #ddd;
}
blockquote>:first-child {
margin-top: 0;
}
blockquote>:last-child {
margin-bottom: 0;
}
table {
display: block;
width: 100%;
overflow: auto;
word-break: normal;
word-break: keep-all;
}
table th {
font-weight: bold;
}
table th,
table td {
padding: 6px 13px;
border: 1px solid #ddd;
}
table tr {
background-color: #fff;
border-top: 1px solid #ccc;
}
table tr:nth-child(2n) {
background-color: #f8f8f8;
}
img {
max-width: 100%;
box-sizing: content-box;
background-color: #fff;
}
code {
padding: 0;
padding-top: 0.2em;
padding-bottom: 0.2em;
margin: 0;
font-size: 85%;
background-color: rgba(0,0,0,0.04);
border-radius: 3px;
}
code:before,
code:after {
letter-spacing: -0.2em;
content: "\00a0";
}
pre>code {
padding: 0;
margin: 0;
font-size: 100%;
word-break: normal;
white-space: pre;
background: transparent;
border: 0;
}
.highlight {
margin-bottom: 16px;
}
.highlight pre,
pre {
padding: 16px;
overflow: auto;
font-size: 85%;
line-height: 1.45;
background-color: #f7f7f7;
border-radius: 3px;
}
.highlight pre {
margin-bottom: 0;
word-break: normal;
}
pre {
word-wrap: normal;
}
pre code {
display: inline;
max-width: initial;
padding: 0;
margin: 0;
overflow: initial;
line-height: inherit;
word-wrap: normal;
background-color: transparent;
border: 0;
}
pre code:before,
pre code:after {
content: normal;
}
kbd {
display: inline-block;
padding: 3px 5px;
font-size: 11px;
line-height: 10px;
color: #555;
vertical-align: middle;
background-color: #fcfcfc;
border: solid 1px #ccc;
border-bottom-color: #bbb;
border-radius: 3px;
box-shadow: inset 0 -1px 0 #bbb;
}
.pl-c {
color: #969896;
}
.pl-c1,
.pl-s .pl-v {
color: #0086b3;
}
.pl-e,
.pl-en {
color: #795da3;
}
.pl-s .pl-s1,
.pl-smi {
color: #333;
}
.pl-ent {
color: #63a35c;
}
.pl-k {
color: #a71d5d;
}
.pl-pds,
.pl-s,
.pl-s .pl-pse .pl-s1,
.pl-sr,
.pl-sr .pl-cce,
.pl-sr .pl-sra,
.pl-sr .pl-sre {
color: #183691;
}
.pl-v {
color: #ed6a43;
}
.pl-id {
color: #b52a1d;
}
.pl-ii {
background-color: #b52a1d;
color: #f8f8f8;
}
.pl-sr .pl-cce {
color: #63a35c;
font-weight: bold;
}
.pl-ml {
color: #693a17;
}
.pl-mh,
.pl-mh .pl-en,
.pl-ms {
color: #1d3e81;
font-weight: bold;
}
.pl-mq {
color: #008080;
}
.pl-mi {
color: #333;
font-style: italic;
}
.pl-mb {
color: #333;
font-weight: bold;
}
.pl-md {
background-color: #ffecec;
color: #bd2c00;
}
.pl-mi1 {
background-color: #eaffea;
color: #55a532;
}
.pl-mdr {
color: #795da3;
font-weight: bold;
}
.pl-mo {
color: #1d3e81;
}
kbd {
display: inline-block;
padding: 3px 5px;
font: 11px Consolas, "Liberation Mono", Menlo, Courier, monospace;
line-height: 10px;
color: #555;
vertical-align: middle;
background-color: #fcfcfc;
border: solid 1px #ccc;
border-bottom-color: #bbb;
border-radius: 3px;
box-shadow: inset 0 -1px 0 #bbb;
}
.task-list-item {
list-style-type: none;
}
.task-list-item+.task-list-item {
margin-top: 3px;
}
.task-list-item input {
margin: 0 0.35em 0.25em -1.6em;
vertical-align: middle;
}
:checked+.radio-label {
z-index: 1;
position: relative;
border-color: #4078c0;
}
.sourceLine {
display: inline-block;
}
code .kw { color: #000000; }
code .dt { color: #ed6a43; }
code .dv { color: #009999; }
code .bn { color: #009999; }
code .fl { color: #009999; }
code .ch { color: #009999; }
code .st { color: #183691; }
code .co { color: #969896; }
code .ot { color: #0086b3; }
code .al { color: #a61717; }
code .fu { color: #63a35c; }
code .er { color: #a61717; background-color: #e3d2d2; }
code .wa { color: #000000; }
code .cn { color: #008080; }
code .sc { color: #008080; }
code .vs { color: #183691; }
code .ss { color: #183691; }
code .im { color: #000000; }
code .va {color: #008080; }
code .cf { color: #000000; }
code .op { color: #000000; }
code .bu { color: #000000; }
code .ex { color: #000000; }
code .pp { color: #999999; }
code .at { color: #008080; }
code .do { color: #969896; }
code .an { color: #008080; }
code .cv { color: #008080; }
code .in { color: #008080; }
</style>
<style>
body {
  box-sizing: border-box;
  min-width: 200px;
  max-width: 980px;
  margin: 0 auto;
  padding: 45px;
  padding-top: 0px;
}
</style>

</head>

<body>

<h1 id="vignettes-of-the-influential">Vignettes of the <code>influential</code></h1>
<h2 id="overview">Overview</h2>
<p>The influential package contains several functions that could be categorized into four groups according to their purpose:</p>
<ul>
<li>Network reconstruction</li>
<li>Calculation of centrality measures</li>
<li>Assessment of the association of centrality measures</li>
<li>Identification of the most <code>influential</code> network nodes</li>
</ul>
<hr />
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" data-line-number="1"><span class="kw">library</span>(influential)</a></code></pre></div>
<h2 id="network-reconstruction">Network reconstruction</h2>
<p>Two functions have been obtained from the <code>igraph</code> package for the reconstruction of networks.</p>
<ul>
<li><h3 id="from-a-data-frame">From a data frame</h3>
<p>In the data frame the first and second columns should be composed of source and target nodes.</p>
<p>A sample appropriate data frame is brought below:</p></li>
</ul>
<table>
<thead>
<tr class="header">
<th align="left">lncRNA</th>
<th align="left">Coexpressed.Gene</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">ADAMTS9-AS2</td>
<td align="left">A2M</td>
</tr>
<tr class="even">
<td align="left">ADAMTS9-AS2</td>
<td align="left">ABCA6</td>
</tr>
<tr class="odd">
<td align="left">ADAMTS9-AS2</td>
<td align="left">ABCA8</td>
</tr>
<tr class="even">
<td align="left">ADAMTS9-AS2</td>
<td align="left">ABCA9</td>
</tr>
<tr class="odd">
<td align="left">ADAMTS9-AS2</td>
<td align="left">ABI3BP</td>
</tr>
<tr class="even">
<td align="left">ADAMTS9-AS2</td>
<td align="left">AC093110.3</td>
</tr>
</tbody>
</table>
<p>This is a co-expression dataset obtained from <a href="https://www.ncbi.nlm.nih.gov/pubmed/31211495">PMID: 31211495</a>.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb2-1" data-line-number="1">MyData &lt;-<span class="st"> </span>coexpression.data        <span class="co"># Preparing the data</span></a>
<a class="sourceLine" id="cb2-2" data-line-number="2"></a>
<a class="sourceLine" id="cb2-3" data-line-number="3">My_graph &lt;-<span class="st"> </span><span class="kw">graph_from_data_frame</span>(<span class="dt">d=</span>MyData)        <span class="co"># Reconstructing the graph</span></a></code></pre></div>
<p>If you look at the class of <code>My_graph</code> you should see that it has an <code>igraph</code> class:</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb3-1" data-line-number="1"><span class="kw">class</span>(My_graph)</a>
<a class="sourceLine" id="cb3-2" data-line-number="2"><span class="co">#&gt; [1] &quot;igraph&quot;</span></a></code></pre></div>
<hr />
<ul>
<li><h3 id="from-an-adjacency-matrix">From an adjacency matrix</h3></li>
</ul>
<p>A sample appropriate data frame is brought below:</p>
<table>
<thead>
<tr class="header">
<th></th>
<th align="right">ADAMTS9-AS2</th>
<th align="right">C8orf34-AS1</th>
<th align="right">CADM3-AS1</th>
<th align="right">FAM83A-AS1</th>
<th align="right">FENDRR</th>
<th align="right">LANCL1-AS1</th>
<th align="right">LINC00092</th>
<th align="right">LINC00467</th>
<th align="right">LINC00857</th>
<th align="right">LINC00891</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>ADAMTS9-AS2</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td>C8orf34-AS1</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td>CADM3-AS1</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
</tr>
<tr class="even">
<td>FAM83A-AS1</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">1</td>
<td align="right">0</td>
</tr>
<tr class="odd">
<td>FENDRR</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
</tr>
<tr class="even">
<td>LANCL1-AS1</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
</tr>
</tbody>
</table>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb4-1" data-line-number="1">MyData &lt;-<span class="st"> </span>coexpression.adjacency        <span class="co"># Preparing the data</span></a>
<a class="sourceLine" id="cb4-2" data-line-number="2"></a>
<a class="sourceLine" id="cb4-3" data-line-number="3">My_graph &lt;-<span class="st"> </span><span class="kw">graph_from_adjacency_matrix</span>(MyData)        <span class="co"># Reconstructing the graph</span></a></code></pre></div>
<hr />
<ul>
<li><h3 id="network-vertices">Network vertices</h3></li>
</ul>
<p>Network vertices (nodes) are required in order to calculate their centrality measures. Thus, before calculation of network centrality measures we need to obtain the name of required network vertices. To this end, we use the <code>V</code> function, which is obtained from the <code>igraph</code> package. However, you may provide a character vector of the name of your desired nodes manually.</p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb5-1" data-line-number="1">MyData &lt;-<span class="st"> </span>coexpression.data        <span class="co"># Preparing the data</span></a>
<a class="sourceLine" id="cb5-2" data-line-number="2"></a>
<a class="sourceLine" id="cb5-3" data-line-number="3">My_graph &lt;-<span class="st"> </span><span class="kw">graph_from_data_frame</span>(MyData)        <span class="co"># Reconstructing the graph</span></a>
<a class="sourceLine" id="cb5-4" data-line-number="4"></a>
<a class="sourceLine" id="cb5-5" data-line-number="5">My_graph_vertices &lt;-<span class="st"> </span><span class="kw">V</span>(My_graph)        <span class="co"># Extracting the vertices</span></a></code></pre></div>
<hr />
<h2 id="calculation-of-centrality-measures">Calculation of centrality measures</h2>
<ul>
<li><h3 id="degree-centrality">Degree centrality</h3>
<p>Degree centrality is the most commonly used local centrality measure which could be calculated via the <code>degree</code> function obtained from the <code>igraph</code> package.</p></li>
</ul>
<!-- end list -->

<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb6-1" data-line-number="1">MyData &lt;-<span class="st"> </span>coexpression.data        <span class="co"># Preparing the data</span></a>
<a class="sourceLine" id="cb6-2" data-line-number="2"></a>
<a class="sourceLine" id="cb6-3" data-line-number="3">My_graph &lt;-<span class="st"> </span><span class="kw">graph_from_data_frame</span>(MyData)        <span class="co"># Reconstructing the graph</span></a>
<a class="sourceLine" id="cb6-4" data-line-number="4"></a>
<a class="sourceLine" id="cb6-5" data-line-number="5">GraphVertices &lt;-<span class="st"> </span><span class="kw">V</span>(My_graph)        <span class="co"># Extracting the vertices</span></a>
<a class="sourceLine" id="cb6-6" data-line-number="6"></a>
<a class="sourceLine" id="cb6-7" data-line-number="7">My_graph_degree &lt;-<span class="st"> </span><span class="kw">degree</span>(My_graph, <span class="dt">v =</span> GraphVertices, <span class="dt">normalized =</span> <span class="ot">FALSE</span>) <span class="co"># Calculating degree centrality</span></a></code></pre></div>
<p>Degree centrality could be also calculated for <em>directed</em> graphs via specifying the <code>mode</code> parameter.</p>
<ul>
<li><h3 id="betweenness-centrality">Betweenness centrality</h3>
<p>Betweenness centrality, like degree centrality, is one of the most commonly used centrality measures but is representative of the global centrality of a node. This centrality metric could also be calculated using a function obtained from the <code>igraph</code> package.</p></li>
</ul>
<!-- end list -->

<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb7-1" data-line-number="1">MyData &lt;-<span class="st"> </span>coexpression.data        <span class="co"># Preparing the data</span></a>
<a class="sourceLine" id="cb7-2" data-line-number="2"></a>
<a class="sourceLine" id="cb7-3" data-line-number="3">My_graph &lt;-<span class="st"> </span><span class="kw">graph_from_data_frame</span>(MyData)        <span class="co"># Reconstructing the graph</span></a>
<a class="sourceLine" id="cb7-4" data-line-number="4"></a>
<a class="sourceLine" id="cb7-5" data-line-number="5">GraphVertices &lt;-<span class="st"> </span><span class="kw">V</span>(My_graph)        <span class="co"># Extracting the vertices</span></a>
<a class="sourceLine" id="cb7-6" data-line-number="6"></a>
<a class="sourceLine" id="cb7-7" data-line-number="7">My_graph_betweenness &lt;-<span class="st"> </span><span class="kw">betweenness</span>(My_graph, <span class="dt">v =</span> GraphVertices,    <span class="co"># Calculating betweenness centrality</span></a>
<a class="sourceLine" id="cb7-8" data-line-number="8">                                    <span class="dt">directed =</span> <span class="ot">FALSE</span>, <span class="dt">normalized =</span> <span class="ot">FALSE</span>)</a></code></pre></div>
<p>Betweenness centrality could be also calculated for <em>directed</em> and/or <em>weighted</em> graphs via specifying the <code>directed</code> and <code>weights</code> parameters, respectively.</p>
<ul>
<li><h3 id="neighborhood-connectivity">Neighborhood connectivity</h3></li>
</ul>
<p>Neighborhood connectivity is one of the other important centrality measures that reflect the semi-local centrality of a node. This centrality measure was first represented in a <a href="https://www.ncbi.nlm.nih.gov/pubmed/11988575">Science paper</a> in 2002 and is for the first time calculable in R environment via the <code>influential</code> package.</p>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb8-1" data-line-number="1">MyData &lt;-<span class="st"> </span>coexpression.data        <span class="co"># Preparing the data</span></a>
<a class="sourceLine" id="cb8-2" data-line-number="2"></a>
<a class="sourceLine" id="cb8-3" data-line-number="3">My_graph &lt;-<span class="st"> </span><span class="kw">graph_from_data_frame</span>(MyData)        <span class="co"># Reconstructing the graph</span></a>
<a class="sourceLine" id="cb8-4" data-line-number="4"></a>
<a class="sourceLine" id="cb8-5" data-line-number="5">GraphVertices &lt;-<span class="st"> </span><span class="kw">V</span>(My_graph)        <span class="co"># Extracting the vertices</span></a>
<a class="sourceLine" id="cb8-6" data-line-number="6"></a>
<a class="sourceLine" id="cb8-7" data-line-number="7">neighrhood.co &lt;-<span class="st"> </span><span class="kw">neighborhood.connectivity</span>(<span class="dt">graph =</span> My_graph,    <span class="co"># Calculating neighborhood connectivity</span></a>
<a class="sourceLine" id="cb8-8" data-line-number="8">                                           <span class="dt">vertices =</span> GraphVertices,</a>
<a class="sourceLine" id="cb8-9" data-line-number="9">                                           <span class="dt">mode =</span> <span class="st">&quot;all&quot;</span>)</a></code></pre></div>
<p>Neighborhood connectivity could be also calculated for <em>directed</em> graphs via specifying the <code>mode</code> parameter.</p>
<ul>
<li><h3 id="h-index">H-index</h3></li>
</ul>
<p>H-index is H-index is another semi-local centrality measure that was inspired from its application in assessing the impact of researchers and is for the first time calculable in R environment via the <code>influential</code> package.</p>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb9-1" data-line-number="1">MyData &lt;-<span class="st"> </span>coexpression.data        <span class="co"># Preparing the data</span></a>
<a class="sourceLine" id="cb9-2" data-line-number="2"></a>
<a class="sourceLine" id="cb9-3" data-line-number="3">My_graph &lt;-<span class="st"> </span><span class="kw">graph_from_data_frame</span>(MyData)        <span class="co"># Reconstructing the graph</span></a>
<a class="sourceLine" id="cb9-4" data-line-number="4"></a>
<a class="sourceLine" id="cb9-5" data-line-number="5">GraphVertices &lt;-<span class="st"> </span><span class="kw">V</span>(My_graph)        <span class="co"># Extracting the vertices</span></a>
<a class="sourceLine" id="cb9-6" data-line-number="6"></a>
<a class="sourceLine" id="cb9-7" data-line-number="7">h.index &lt;-<span class="st"> </span><span class="kw">h_index</span>(<span class="dt">graph =</span> My_graph,    <span class="co"># Calculating H-index</span></a>
<a class="sourceLine" id="cb9-8" data-line-number="8">                   <span class="dt">vertices =</span> GraphVertices,</a>
<a class="sourceLine" id="cb9-9" data-line-number="9">                   <span class="dt">mode =</span> <span class="st">&quot;all&quot;</span>)</a></code></pre></div>
<p>H-index could be also calculated for <em>directed</em> graphs via specifying the <code>mode</code> parameter.</p>
<ul>
<li><h3 id="local-h-index">Local H-index</h3></li>
</ul>
<p>Local H-index (LH-index) is a semi-local centrality measure and an improved version of H-index centrality that leverages the H-index to the second order neighbors of a node and is for the first time calculable in R environment via the <code>influential</code> package.</p>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb10-1" data-line-number="1">MyData &lt;-<span class="st"> </span>coexpression.data        <span class="co"># Preparing the data</span></a>
<a class="sourceLine" id="cb10-2" data-line-number="2"></a>
<a class="sourceLine" id="cb10-3" data-line-number="3">My_graph &lt;-<span class="st"> </span><span class="kw">graph_from_data_frame</span>(MyData)        <span class="co"># Reconstructing the graph</span></a>
<a class="sourceLine" id="cb10-4" data-line-number="4"></a>
<a class="sourceLine" id="cb10-5" data-line-number="5">GraphVertices &lt;-<span class="st"> </span><span class="kw">V</span>(My_graph)        <span class="co"># Extracting the vertices</span></a>
<a class="sourceLine" id="cb10-6" data-line-number="6"></a>
<a class="sourceLine" id="cb10-7" data-line-number="7">lh.index &lt;-<span class="st"> </span><span class="kw">lh_index</span>(<span class="dt">graph =</span> My_graph,    <span class="co"># Calculating Local H-index</span></a>
<a class="sourceLine" id="cb10-8" data-line-number="8">                   <span class="dt">vertices =</span> GraphVertices,</a>
<a class="sourceLine" id="cb10-9" data-line-number="9">                   <span class="dt">mode =</span> <span class="st">&quot;all&quot;</span>)</a></code></pre></div>
<p>Local H-index could be also calculated for <em>directed</em> graphs via specifying the <code>mode</code> parameter.</p>
<ul>
<li><h3 id="collective-influence">Collective Influence</h3></li>
</ul>
<p>Collective Influence (CI) is a global centrality measure that calculates the product of the reduced degree (degree - 1) of a node and the total reduced degree of all nodes at a distance d from the node. This centrality measure is for the first time provided in an R package.</p>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb11-1" data-line-number="1">MyData &lt;-<span class="st"> </span>coexpression.data        <span class="co"># Preparing the data</span></a>
<a class="sourceLine" id="cb11-2" data-line-number="2"></a>
<a class="sourceLine" id="cb11-3" data-line-number="3">My_graph &lt;-<span class="st"> </span><span class="kw">graph_from_data_frame</span>(MyData)        <span class="co"># Reconstructing the graph</span></a>
<a class="sourceLine" id="cb11-4" data-line-number="4"></a>
<a class="sourceLine" id="cb11-5" data-line-number="5">GraphVertices &lt;-<span class="st"> </span><span class="kw">V</span>(My_graph)        <span class="co"># Extracting the vertices</span></a>
<a class="sourceLine" id="cb11-6" data-line-number="6"></a>
<a class="sourceLine" id="cb11-7" data-line-number="7">ci &lt;-<span class="st"> </span><span class="kw">collective.influence</span>(<span class="dt">graph =</span> My_graph,    <span class="co"># Calculating Collective Influence</span></a>
<a class="sourceLine" id="cb11-8" data-line-number="8">                          <span class="dt">vertices =</span> GraphVertices,</a>
<a class="sourceLine" id="cb11-9" data-line-number="9">                          <span class="dt">mode =</span> <span class="st">&quot;all&quot;</span>, <span class="dt">d=</span><span class="dv">3</span>)</a></code></pre></div>
<p>Collective Influence could be also calculated for <em>directed</em> graphs via specifying the <code>mode</code> parameter.</p>
<ul>
<li><h3 id="clusterrank">ClusterRank</h3></li>
</ul>
<p>ClusterRank is a local centrality measure that makes a connection between local and semi-local characteristics of a node and at the same time removes the negative effects of local clustering.</p>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb12-1" data-line-number="1">MyData &lt;-<span class="st"> </span>coexpression.data        <span class="co"># Preparing the data</span></a>
<a class="sourceLine" id="cb12-2" data-line-number="2"></a>
<a class="sourceLine" id="cb12-3" data-line-number="3">My_graph &lt;-<span class="st"> </span><span class="kw">graph_from_data_frame</span>(MyData)        <span class="co"># Reconstructing the graph</span></a>
<a class="sourceLine" id="cb12-4" data-line-number="4"></a>
<a class="sourceLine" id="cb12-5" data-line-number="5">GraphVertices &lt;-<span class="st"> </span><span class="kw">V</span>(My_graph)        <span class="co"># Extracting the vertices</span></a>
<a class="sourceLine" id="cb12-6" data-line-number="6"></a>
<a class="sourceLine" id="cb12-7" data-line-number="7">cr &lt;-<span class="st"> </span><span class="kw">clusterRank</span>(<span class="dt">graph =</span> My_graph,    <span class="co"># Calculating ClusterRank</span></a>
<a class="sourceLine" id="cb12-8" data-line-number="8">                  <span class="dt">vids =</span> GraphVertices,</a>
<a class="sourceLine" id="cb12-9" data-line-number="9">                  <span class="dt">directed =</span> <span class="ot">FALSE</span>, <span class="dt">loops =</span> <span class="ot">TRUE</span>)</a></code></pre></div>
<p>ClusterRank could be also calculated for <em>directed</em> graphs via specifying the <code>directed</code> parameter.</p>
<hr />
<h2 id="assessment-of-the-association-of-centrality-measures">Assessment of the association of centrality measures</h2>
<ul>
<li><h3 id="conditional-probability-of-deviation-from-means">Conditional probability of deviation from means</h3></li>
</ul>
<p>The function <code>cond.prob.analysis</code> assesses the conditional probability of deviation of two centrality measures (or any other two continuous variables) from their corresponding means in opposite directions.</p>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb13-1" data-line-number="1">MyData &lt;-<span class="st"> </span>centrality.measures        <span class="co"># Preparing the data</span></a>
<a class="sourceLine" id="cb13-2" data-line-number="2"></a>
<a class="sourceLine" id="cb13-3" data-line-number="3">My.conditional.prob &lt;-<span class="st"> </span><span class="kw">cond.prob.analysis</span>(<span class="dt">data =</span> MyData,       <span class="co"># Assessing the conditional probability</span></a>
<a class="sourceLine" id="cb13-4" data-line-number="4">                                          <span class="dt">nodes.colname =</span> <span class="kw">rownames</span>(MyData),</a>
<a class="sourceLine" id="cb13-5" data-line-number="5">                                          <span class="dt">Desired.colname =</span> <span class="st">&quot;BC&quot;</span>,</a>
<a class="sourceLine" id="cb13-6" data-line-number="6">                                          <span class="dt">Condition.colname =</span> <span class="st">&quot;NC&quot;</span>)</a>
<a class="sourceLine" id="cb13-7" data-line-number="7"></a>
<a class="sourceLine" id="cb13-8" data-line-number="8"><span class="kw">print</span>(My.conditional.prob)</a>
<a class="sourceLine" id="cb13-9" data-line-number="9"><span class="co">#&gt; $ConditionalProbability</span></a>
<a class="sourceLine" id="cb13-10" data-line-number="10"><span class="co">#&gt; [1] 51.61871</span></a>
<a class="sourceLine" id="cb13-11" data-line-number="11"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb13-12" data-line-number="12"><span class="co">#&gt; $ConditionalProbability_split.half.sample</span></a>
<a class="sourceLine" id="cb13-13" data-line-number="13"><span class="co">#&gt; [1] 51.33333</span></a></code></pre></div>
<ul>
<li>As you can see in the results, the whole data is also randomly splitted into half in order to further test the validity of conditional probability assessments.</li>
<li><em>The higher the conditional probability the more two centrality measures behave in contrary manners</em>.</li>
</ul>
<hr />
<ul>
<li><h3 id="nature-of-association-(considering-dependent-and-independent)">Nature of association (considering dependent and independent)</h3></li>
</ul>
<p>The function <code>double.cent.assess</code> could be used to automatically assess both the distribution mode of centrality measures (two continuous variables) and the nature of their association. The analyses done through this formula are as follows:</p>
<ol>
<li><strong>Normality assessment</strong>:
<ul>
<li>Variables with <strong>lower than</strong> 5000 observations: <em>Shapiro-Wilk test</em></li>
<li>Variables with <strong>over</strong> 5000 observations: <em>Anderson-Darling test</em> <br><br></li>
</ul></li>
<li><strong>Assessment of non-linear/non-monotonic correlation</strong>:
<ul>
<li><em>Non-linearity assessment</em>: Fitting a generalized additive model (GAM) with integrated smoothness approximations using the <code>mgcv</code> package <br><br></li>
<li><em>Non-monotonicity assessment</em>: Comparing the squared coefficients of the correlation based on Spearman’s rank correlation analysis and ranked regression test with non-linear splines.
<ul>
<li>Squared coefficient of Spearman’s rank correlation <strong>&gt;</strong> R-squared ranked regression with non-linear splines: <em>Monotonic</em></li>
<li>Squared coefficient of Spearman’s rank correlation <strong>&lt;</strong> R-squared ranked regression with non-linear splines: <em>Non-monotonic</em> <br><br></li>
</ul></li>
</ul></li>
<li><strong>Dependence assessment</strong>:
<ul>
<li><em>Hoeffding’s independence test</em>: Hoeffding’s test of independence is a test based on the population measure of deviation from independence which computes a D Statistics ranging from -0.5 to 1: Greater D values indicate a higher dependence between variables.</li>
<li><em>Descriptive non-linear non-parametric dependence test</em>: This assessment is based on non-linear non-parametric statistics (NNS) which outputs a dependence value ranging from 0 to 1. For further details please refer to <a href="https://cran.r-project.org/package=NNS">NNS: Nonlinear Nonparametric Statistics</a>: Greater values indicate a higher dependence between variables. <br><br></li>
</ul></li>
<li><strong>Correlation assessment</strong>: As the correlation between most of the centrality measures follows a non-monotonic form, this part of the assessment is done based on the non-linear non-parametric statistics (NNS) which itself calculates the correlation based on partial moments and outputs a correlation value ranging from -1 to 1. For further details please refer to <a href="https://cran.r-project.org/package=NNS">NNS: Nonlinear Nonparametric Statistics</a>. <br><br></li>
<li><strong>Assessment of conditional probability of deviation from means</strong> This step assesses the conditional probability of deviation of two centrality measures (or any other two continuous variables) from their corresponding means in opposite directions.
<ul>
<li>The independent centrality measure (variable) is considered as the condition variable and the other as the desired one.</li>
<li>As you will see in the results, the whole data is also randomly splitted into half in order to further test the validity of conditional probability assessments.</li>
<li><em>The higher the conditional probability the more two centrality measures behave in contrary manners</em>.</li>
</ul></li>
</ol>
<!-- end list -->

<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb14-1" data-line-number="1">MyData &lt;-<span class="st"> </span>centrality.measures        <span class="co"># Preparing the data</span></a>
<a class="sourceLine" id="cb14-2" data-line-number="2"></a>
<a class="sourceLine" id="cb14-3" data-line-number="3">My.metrics.assessment &lt;-<span class="st"> </span><span class="kw">double.cent.assess</span>(<span class="dt">data =</span> MyData,       <span class="co"># Association assessment</span></a>
<a class="sourceLine" id="cb14-4" data-line-number="4">                                            <span class="dt">nodes.colname =</span> <span class="kw">rownames</span>(MyData),</a>
<a class="sourceLine" id="cb14-5" data-line-number="5">                                            <span class="dt">dependent.colname =</span> <span class="st">&quot;BC&quot;</span>,</a>
<a class="sourceLine" id="cb14-6" data-line-number="6">                                            <span class="dt">independent.colname =</span> <span class="st">&quot;NC&quot;</span>)</a>
<a class="sourceLine" id="cb14-7" data-line-number="7"></a>
<a class="sourceLine" id="cb14-8" data-line-number="8"><span class="kw">print</span>(My.metrics.assessment)</a>
<a class="sourceLine" id="cb14-9" data-line-number="9"><span class="co">#&gt; $Summary_statistics</span></a>
<a class="sourceLine" id="cb14-10" data-line-number="10"><span class="co">#&gt;         BC NC</span></a>
<a class="sourceLine" id="cb14-11" data-line-number="11"><span class="co">#&gt; Min.              0.000000000                   1.2000</span></a>
<a class="sourceLine" id="cb14-12" data-line-number="12"><span class="co">#&gt; 1st Qu.           0.000000000                  66.0000</span></a>
<a class="sourceLine" id="cb14-13" data-line-number="13"><span class="co">#&gt; Median            0.000000000                 156.0000</span></a>
<a class="sourceLine" id="cb14-14" data-line-number="14"><span class="co">#&gt; Mean              0.005813357                 132.3443</span></a>
<a class="sourceLine" id="cb14-15" data-line-number="15"><span class="co">#&gt; 3rd Qu.           0.000340000                 179.3214</span></a>
<a class="sourceLine" id="cb14-16" data-line-number="16"><span class="co">#&gt; Max.              0.529464720                 192.0000</span></a>
<a class="sourceLine" id="cb14-17" data-line-number="17"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb14-18" data-line-number="18"><span class="co">#&gt; $Normality_results</span></a>
<a class="sourceLine" id="cb14-19" data-line-number="19"><span class="co">#&gt;                               p.value</span></a>
<a class="sourceLine" id="cb14-20" data-line-number="20"><span class="co">#&gt; BC    1.415450e-50</span></a>
<a class="sourceLine" id="cb14-21" data-line-number="21"><span class="co">#&gt; NC 9.411737e-30</span></a>
<a class="sourceLine" id="cb14-22" data-line-number="22"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb14-23" data-line-number="23"><span class="co">#&gt; $Dependent_Normality</span></a>
<a class="sourceLine" id="cb14-24" data-line-number="24"><span class="co">#&gt; [1] &quot;Non-normally distributed&quot;</span></a>
<a class="sourceLine" id="cb14-25" data-line-number="25"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb14-26" data-line-number="26"><span class="co">#&gt; $Independent_Normality</span></a>
<a class="sourceLine" id="cb14-27" data-line-number="27"><span class="co">#&gt; [1] &quot;Non-normally distributed&quot;</span></a>
<a class="sourceLine" id="cb14-28" data-line-number="28"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb14-29" data-line-number="29"><span class="co">#&gt; $GAM_nonlinear.nonmonotonic.results</span></a>
<a class="sourceLine" id="cb14-30" data-line-number="30"><span class="co">#&gt;      edf  p-value </span></a>
<a class="sourceLine" id="cb14-31" data-line-number="31"><span class="co">#&gt; 8.992406 0.000000 </span></a>
<a class="sourceLine" id="cb14-32" data-line-number="32"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb14-33" data-line-number="33"><span class="co">#&gt; $Association_type</span></a>
<a class="sourceLine" id="cb14-34" data-line-number="34"><span class="co">#&gt; [1] &quot;nonlinear-nonmonotonic&quot;</span></a>
<a class="sourceLine" id="cb14-35" data-line-number="35"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb14-36" data-line-number="36"><span class="co">#&gt; $HoeffdingD_Statistic</span></a>
<a class="sourceLine" id="cb14-37" data-line-number="37"><span class="co">#&gt;         D_statistic P_value</span></a>
<a class="sourceLine" id="cb14-38" data-line-number="38"><span class="co">#&gt; Results  0.01770279   1e-08</span></a>
<a class="sourceLine" id="cb14-39" data-line-number="39"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb14-40" data-line-number="40"><span class="co">#&gt; $Dependence_Significance</span></a>
<a class="sourceLine" id="cb14-41" data-line-number="41"><span class="co">#&gt;                       Hoeffding</span></a>
<a class="sourceLine" id="cb14-42" data-line-number="42"><span class="co">#&gt; Results Significantly dependent</span></a>
<a class="sourceLine" id="cb14-43" data-line-number="43"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb14-44" data-line-number="44"><span class="co">#&gt; $NNS_dep_results</span></a>
<a class="sourceLine" id="cb14-45" data-line-number="45"><span class="co">#&gt;         Correlation Dependence</span></a>
<a class="sourceLine" id="cb14-46" data-line-number="46"><span class="co">#&gt; Results  -0.7948106  0.8647164</span></a>
<a class="sourceLine" id="cb14-47" data-line-number="47"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb14-48" data-line-number="48"><span class="co">#&gt; $ConditionalProbability</span></a>
<a class="sourceLine" id="cb14-49" data-line-number="49"><span class="co">#&gt; [1] 55.35386</span></a>
<a class="sourceLine" id="cb14-50" data-line-number="50"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb14-51" data-line-number="51"><span class="co">#&gt; $ConditionalProbability_split.half.sample</span></a>
<a class="sourceLine" id="cb14-52" data-line-number="52"><span class="co">#&gt; [1] 55.90331</span></a></code></pre></div>
<p><strong>Note</strong>: It should also be noted that as a single regression line does not fit all models with a certain degree of freedom, based on the size and correlation mode of the variables provided, this function might return an error due to incapability of running step 2. In this case, you may follow each step manually or as an alternative run the other function named <code>double.cent.assess.noRegression</code> which does not perform any regression test and consequently it is not required to determine the dependent and independent variables.</p>
<hr />
<ul>
<li><h3 id="nature-of-association-(without-considering-dependence-direction)">Nature of association (without considering dependence direction)</h3></li>
</ul>
<p>The function <code>double.cent.assess.noRegression</code> could be used to automatically assess both the distribution mode of centrality measures (two continuous variables) and the nature of their association. The analyses done through this formula are as follows:</p>
<ol>
<li><strong>Normality assessment</strong>:
<ul>
<li>Variables with <strong>lower than</strong> 5000 observations: <em>Shapiro-Wilk test</em></li>
<li>Variables with <strong>over</strong> 5000 observations: <em>Anderson–Darling test</em> <br><br></li>
</ul></li>
<li><strong>Dependence assessment</strong>:
<ul>
<li><em>Hoeffding’s independence test</em>: Hoeffding’s test of independence is a test based on the population measure of deviation from independence which computes a D Statistics ranging from -0.5 to 1: Greater D values indicate a higher dependence between variables.</li>
<li><em>Descriptive non-linear non-parametric dependence test</em>: This assessment is based on non-linear non-parametric statistics (NNS) which outputs a dependence value ranging from 0 to 1. For further details please refer to <a href="https://cran.r-project.org/package=NNS">NNS: Nonlinear Nonparametric Statistics</a>: Greater values indicate a higher dependence between variables. <br><br></li>
</ul></li>
<li><strong>Correlation assessment</strong>: As the correlation between most of the centrality measures follows a non-monotonic form, this part of the assessment is done based on the non-linear non-parametric statistics (NNS) which itself calculates the correlation based on partial moments and outputs a correlation value ranging from -1 to 1. For further details please refer to <a href="https://cran.r-project.org/package=NNS">NNS: Nonlinear Nonparametric Statistics</a>. <br><br></li>
<li><strong>Assessment of conditional probability of deviation from means</strong> This step assesses the conditional probability of deviation of two centrality measures (or any other two continuous variables) from their corresponding means in opposite directions.
<ul>
<li>The <code>centrality2</code> variable is considered as the condition variable and the other (<code>centrality1</code>) as the desired one.</li>
<li>As you will see in the results, the whole data is also randomly splitted into half in order to further test the validity of conditional probability assessments.</li>
<li><em>The higher the conditional probability the more two centrality measures behave in contrary manners</em>.</li>
</ul></li>
</ol>
<!-- end list -->

<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb15-1" data-line-number="1">MyData &lt;-<span class="st"> </span>centrality.measures        <span class="co"># Preparing the data</span></a>
<a class="sourceLine" id="cb15-2" data-line-number="2"></a>
<a class="sourceLine" id="cb15-3" data-line-number="3">My.metrics.assessment &lt;-<span class="st"> </span><span class="kw">double.cent.assess.noRegression</span>(<span class="dt">data =</span> MyData,       <span class="co"># Association assessment</span></a>
<a class="sourceLine" id="cb15-4" data-line-number="4">                                                         <span class="dt">nodes.colname =</span> <span class="kw">rownames</span>(MyData),</a>
<a class="sourceLine" id="cb15-5" data-line-number="5">                                                         <span class="dt">centrality1.colname =</span> <span class="st">&quot;BC&quot;</span>,</a>
<a class="sourceLine" id="cb15-6" data-line-number="6">                                                         <span class="dt">centrality2.colname =</span> <span class="st">&quot;NC&quot;</span>)</a>
<a class="sourceLine" id="cb15-7" data-line-number="7"></a>
<a class="sourceLine" id="cb15-8" data-line-number="8"><span class="kw">print</span>(My.metrics.assessment)</a>
<a class="sourceLine" id="cb15-9" data-line-number="9"><span class="co">#&gt; $Summary_statistics</span></a>
<a class="sourceLine" id="cb15-10" data-line-number="10"><span class="co">#&gt;         BC NC</span></a>
<a class="sourceLine" id="cb15-11" data-line-number="11"><span class="co">#&gt; Min.              0.000000000                   1.2000</span></a>
<a class="sourceLine" id="cb15-12" data-line-number="12"><span class="co">#&gt; 1st Qu.           0.000000000                  66.0000</span></a>
<a class="sourceLine" id="cb15-13" data-line-number="13"><span class="co">#&gt; Median            0.000000000                 156.0000</span></a>
<a class="sourceLine" id="cb15-14" data-line-number="14"><span class="co">#&gt; Mean              0.005813357                 132.3443</span></a>
<a class="sourceLine" id="cb15-15" data-line-number="15"><span class="co">#&gt; 3rd Qu.           0.000340000                 179.3214</span></a>
<a class="sourceLine" id="cb15-16" data-line-number="16"><span class="co">#&gt; Max.              0.529464720                 192.0000</span></a>
<a class="sourceLine" id="cb15-17" data-line-number="17"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb15-18" data-line-number="18"><span class="co">#&gt; $Normality_results</span></a>
<a class="sourceLine" id="cb15-19" data-line-number="19"><span class="co">#&gt;                               p.value</span></a>
<a class="sourceLine" id="cb15-20" data-line-number="20"><span class="co">#&gt; BC    1.415450e-50</span></a>
<a class="sourceLine" id="cb15-21" data-line-number="21"><span class="co">#&gt; NC 9.411737e-30</span></a>
<a class="sourceLine" id="cb15-22" data-line-number="22"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb15-23" data-line-number="23"><span class="co">#&gt; $Centrality1_Normality</span></a>
<a class="sourceLine" id="cb15-24" data-line-number="24"><span class="co">#&gt; [1] &quot;Non-normally distributed&quot;</span></a>
<a class="sourceLine" id="cb15-25" data-line-number="25"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb15-26" data-line-number="26"><span class="co">#&gt; $Centrality2_Normality</span></a>
<a class="sourceLine" id="cb15-27" data-line-number="27"><span class="co">#&gt; [1] &quot;Non-normally distributed&quot;</span></a>
<a class="sourceLine" id="cb15-28" data-line-number="28"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb15-29" data-line-number="29"><span class="co">#&gt; $HoeffdingD_Statistic</span></a>
<a class="sourceLine" id="cb15-30" data-line-number="30"><span class="co">#&gt;         D_statistic P_value</span></a>
<a class="sourceLine" id="cb15-31" data-line-number="31"><span class="co">#&gt; Results  0.01770279   1e-08</span></a>
<a class="sourceLine" id="cb15-32" data-line-number="32"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb15-33" data-line-number="33"><span class="co">#&gt; $Dependence_Significance</span></a>
<a class="sourceLine" id="cb15-34" data-line-number="34"><span class="co">#&gt;                       Hoeffding</span></a>
<a class="sourceLine" id="cb15-35" data-line-number="35"><span class="co">#&gt; Results Significantly dependent</span></a>
<a class="sourceLine" id="cb15-36" data-line-number="36"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb15-37" data-line-number="37"><span class="co">#&gt; $NNS_dep_results</span></a>
<a class="sourceLine" id="cb15-38" data-line-number="38"><span class="co">#&gt;         Correlation Dependence</span></a>
<a class="sourceLine" id="cb15-39" data-line-number="39"><span class="co">#&gt; Results  -0.7948106  0.8647164</span></a>
<a class="sourceLine" id="cb15-40" data-line-number="40"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb15-41" data-line-number="41"><span class="co">#&gt; $ConditionalProbability</span></a>
<a class="sourceLine" id="cb15-42" data-line-number="42"><span class="co">#&gt; [1] 55.35386</span></a>
<a class="sourceLine" id="cb15-43" data-line-number="43"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb15-44" data-line-number="44"><span class="co">#&gt; $ConditionalProbability_split.half.sample</span></a>
<a class="sourceLine" id="cb15-45" data-line-number="45"><span class="co">#&gt; [1] 55.68163</span></a></code></pre></div>
<hr />
<h2 id="identification-of-the-most-influential-network-nodes">Identification of the most <code>influential</code> network nodes</h2>
<p><strong>IVI</strong> : <code>IVI</code> is the first integrative method for the identification of network most influential nodes in a way that captures all network topological dimensions. The <code>IVI</code> formula integrates the most important local (i.e. degree centrality and ClusterRank), semi-local (i.e. neighborhood connectivity and local H-index) and global (i.e. betweenness centrality and collective influence) centrality measures in such a way that both synergize their effects and remove their biases.</p>
<ul>
<li><h3 id="integrated-value-of-influence-(ivi)-from-centrality-measures">Integrated Value of Influence (IVI) from centrality measures</h3></li>
</ul>
<!-- end list -->

<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb16-1" data-line-number="1">MyData &lt;-<span class="st"> </span>centrality.measures        <span class="co"># Preparing the data</span></a>
<a class="sourceLine" id="cb16-2" data-line-number="2"></a>
<a class="sourceLine" id="cb16-3" data-line-number="3">My.vertices.IVI &lt;-<span class="st"> </span><span class="kw">ivi.from.indices</span>(<span class="dt">DC =</span> centrality.measures<span class="op">$</span>DC,       <span class="co"># Calculation of IVI</span></a>
<a class="sourceLine" id="cb16-4" data-line-number="4">                                   <span class="dt">CR =</span> centrality.measures<span class="op">$</span>CR,</a>
<a class="sourceLine" id="cb16-5" data-line-number="5">                                   <span class="dt">NC =</span> centrality.measures<span class="op">$</span>NC,</a>
<a class="sourceLine" id="cb16-6" data-line-number="6">                                   <span class="dt">LH_index =</span> centrality.measures<span class="op">$</span>LH_index,</a>
<a class="sourceLine" id="cb16-7" data-line-number="7">                                   <span class="dt">BC =</span> centrality.measures<span class="op">$</span>BC,</a>
<a class="sourceLine" id="cb16-8" data-line-number="8">                                   <span class="dt">CI =</span> centrality.measures<span class="op">$</span>CI)</a></code></pre></div>
<ul>
<li><h3 id="integrated-value-of-influence-(ivi)-from-graph">Integrated Value of Influence (IVI) from graph</h3></li>
</ul>
<!-- end list -->

<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb17-1" data-line-number="1">MyData &lt;-<span class="st"> </span>coexpression.data        <span class="co"># Preparing the data</span></a>
<a class="sourceLine" id="cb17-2" data-line-number="2"></a>
<a class="sourceLine" id="cb17-3" data-line-number="3">My_graph &lt;-<span class="st"> </span><span class="kw">graph_from_data_frame</span>(MyData)        <span class="co"># Reconstructing the graph</span></a>
<a class="sourceLine" id="cb17-4" data-line-number="4"></a>
<a class="sourceLine" id="cb17-5" data-line-number="5">GraphVertices &lt;-<span class="st"> </span><span class="kw">V</span>(My_graph)        <span class="co"># Extracting the vertices</span></a>
<a class="sourceLine" id="cb17-6" data-line-number="6"></a>
<a class="sourceLine" id="cb17-7" data-line-number="7">My.vertices.IVI &lt;-<span class="st"> </span><span class="kw">ivi</span>(<span class="dt">graph =</span> My_graph, <span class="dt">vertices =</span> GraphVertices, <span class="co"># Calculation of IVI</span></a>
<a class="sourceLine" id="cb17-8" data-line-number="8">                       <span class="dt">weights =</span> <span class="ot">NULL</span>, <span class="dt">directed =</span> <span class="ot">FALSE</span>, <span class="dt">mode =</span> <span class="st">&quot;all&quot;</span>,</a>
<a class="sourceLine" id="cb17-9" data-line-number="9">                       <span class="dt">loops =</span> <span class="ot">TRUE</span>, <span class="dt">d =</span> <span class="dv">3</span>, <span class="dt">scaled =</span> <span class="ot">TRUE</span>)</a></code></pre></div>
<p>IVI could be also calculated for <em>directed</em> and/or <em>weighted</em> graphs via specifying the <code>directed</code>, <code>mode</code>, and <code>weights</code> parameters.</p>
<hr />
<h2 id="identification-of-the-most-important-network-spreaders">Identification of the most important network spreaders</h2>
<p><strong>Spreading score</strong> : <code>spreading.score</code> is an integrative score made up of four different centrality measures including ClusterRank, neighborhood connectivity, betweenness centrality, and collective influence. Also, Spreading score reflects the spreading potential of each node within a network and is one of the major components of the <code>IVI</code>.</p>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb18-1" data-line-number="1">MyData &lt;-<span class="st"> </span>coexpression.data        <span class="co"># Preparing the data</span></a>
<a class="sourceLine" id="cb18-2" data-line-number="2"></a>
<a class="sourceLine" id="cb18-3" data-line-number="3">My_graph &lt;-<span class="st"> </span><span class="kw">graph_from_data_frame</span>(MyData)        <span class="co"># Reconstructing the graph</span></a>
<a class="sourceLine" id="cb18-4" data-line-number="4"></a>
<a class="sourceLine" id="cb18-5" data-line-number="5">GraphVertices &lt;-<span class="st"> </span><span class="kw">V</span>(My_graph)        <span class="co"># Extracting the vertices</span></a>
<a class="sourceLine" id="cb18-6" data-line-number="6"></a>
<a class="sourceLine" id="cb18-7" data-line-number="7">Spreading.score &lt;-<span class="st"> </span><span class="kw">spreading.score</span>(<span class="dt">graph =</span> My_graph,     <span class="co"># Calculation of Spreading score</span></a>
<a class="sourceLine" id="cb18-8" data-line-number="8">                                   <span class="dt">vertices =</span> GraphVertices, </a>
<a class="sourceLine" id="cb18-9" data-line-number="9">                                   <span class="dt">weights =</span> <span class="ot">NULL</span>, <span class="dt">directed =</span> <span class="ot">FALSE</span>, <span class="dt">mode =</span> <span class="st">&quot;all&quot;</span>,</a>
<a class="sourceLine" id="cb18-10" data-line-number="10">                                   <span class="dt">loops =</span> <span class="ot">TRUE</span>, <span class="dt">d =</span> <span class="dv">3</span>, <span class="dt">scaled =</span> <span class="ot">TRUE</span>)</a></code></pre></div>
<p>Spreading score could be also calculated for <em>directed</em> and/or <em>weighted</em> graphs via specifying the <code>directed</code>, <code>mode</code>, and <code>weights</code> parameters.</p>
<hr />
<h2 id="identification-of-the-most-important-network-hubs">Identification of the most important network hubs</h2>
<p><strong>Hubness score</strong> : <code>hubness.score</code> is an integrative score made up of two different centrality measures including degree centrality and local H-index. Also, Hubness score reflects the power of each node in its surrounding environment and is one of the major components of the <code>IVI</code>.</p>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb19-1" data-line-number="1">MyData &lt;-<span class="st"> </span>coexpression.data        <span class="co"># Preparing the data</span></a>
<a class="sourceLine" id="cb19-2" data-line-number="2"></a>
<a class="sourceLine" id="cb19-3" data-line-number="3">My_graph &lt;-<span class="st"> </span><span class="kw">graph_from_data_frame</span>(MyData)        <span class="co"># Reconstructing the graph</span></a>
<a class="sourceLine" id="cb19-4" data-line-number="4"></a>
<a class="sourceLine" id="cb19-5" data-line-number="5">GraphVertices &lt;-<span class="st"> </span><span class="kw">V</span>(My_graph)        <span class="co"># Extracting the vertices</span></a>
<a class="sourceLine" id="cb19-6" data-line-number="6"></a>
<a class="sourceLine" id="cb19-7" data-line-number="7">Hubness.score &lt;-<span class="st"> </span><span class="kw">hubness.score</span>(<span class="dt">graph =</span> My_graph,     <span class="co"># Calculation of Hubness score</span></a>
<a class="sourceLine" id="cb19-8" data-line-number="8">                                   <span class="dt">vertices =</span> GraphVertices, </a>
<a class="sourceLine" id="cb19-9" data-line-number="9">                                   <span class="dt">directed =</span> <span class="ot">FALSE</span>, <span class="dt">mode =</span> <span class="st">&quot;all&quot;</span>,</a>
<a class="sourceLine" id="cb19-10" data-line-number="10">                                   <span class="dt">loops =</span> <span class="ot">TRUE</span>, <span class="dt">scaled =</span> <span class="ot">TRUE</span>)</a></code></pre></div>
<p>Spreading score could be also calculated for <em>directed</em> graphs via specifying the <code>directed</code> and <code>mode</code> parameters.</p>
<hr />
<h2 id="ranking-the-influence-of-network-nodes-based-on-the-sirir-model">Ranking the influence of network nodes based on the <code>SIRIR</code> model</h2>
<p><strong>SIRIR</strong> : <code>SIRIR</code> is achieved by the integration susceptible-infected-recovered (SIR) model with the leave-one-out cross validation technique and ranks network nodes based on their true universal influence. One of the applications of this function is the assessment of performance of a novel algorithm in identification of network influential nodes.</p>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb20-1" data-line-number="1"><span class="kw">set.seed</span>(<span class="dv">1234</span>)</a>
<a class="sourceLine" id="cb20-2" data-line-number="2">My_graph &lt;-<span class="st"> </span>igraph<span class="op">::</span><span class="kw">sample_gnp</span>(<span class="dt">n=</span><span class="dv">50</span>, <span class="dt">p=</span><span class="fl">0.05</span>)        <span class="co"># Reconstructing the graph</span></a>
<a class="sourceLine" id="cb20-3" data-line-number="3"></a>
<a class="sourceLine" id="cb20-4" data-line-number="4">GraphVertices &lt;-<span class="st"> </span><span class="kw">V</span>(My_graph)        <span class="co"># Extracting the vertices</span></a>
<a class="sourceLine" id="cb20-5" data-line-number="5"></a>
<a class="sourceLine" id="cb20-6" data-line-number="6">Influence.Ranks &lt;-<span class="st"> </span><span class="kw">sirir</span>(<span class="dt">graph =</span> My_graph,     <span class="co"># Calculation of influence rank</span></a>
<a class="sourceLine" id="cb20-7" data-line-number="7">                                   <span class="dt">vertices =</span> GraphVertices, </a>
<a class="sourceLine" id="cb20-8" data-line-number="8">                                   <span class="dt">beta =</span> <span class="fl">0.5</span>, <span class="dt">gamma =</span> <span class="dv">1</span>, <span class="dt">no.sim =</span> <span class="dv">10</span>, <span class="dt">seed =</span> <span class="dv">1234</span>)</a>
<a class="sourceLine" id="cb20-9" data-line-number="9"></a>
<a class="sourceLine" id="cb20-10" data-line-number="10">knitr<span class="op">::</span><span class="kw">kable</span>(Influence.Ranks[<span class="kw">c</span>(<span class="kw">order</span>(Influence.Ranks<span class="op">$</span>rank)[<span class="dv">1</span><span class="op">:</span><span class="dv">10</span>]),])</a></code></pre></div>
<table>
<thead>
<tr class="header">
<th></th>
<th align="right">difference.value</th>
<th align="right">rank</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>6</td>
<td align="right">9.0</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td>1</td>
<td align="right">8.9</td>
<td align="right">2</td>
</tr>
<tr class="odd">
<td>2</td>
<td align="right">8.9</td>
<td align="right">2</td>
</tr>
<tr class="even">
<td>8</td>
<td align="right">8.9</td>
<td align="right">2</td>
</tr>
<tr class="odd">
<td>10</td>
<td align="right">8.7</td>
<td align="right">5</td>
</tr>
<tr class="even">
<td>24</td>
<td align="right">8.7</td>
<td align="right">5</td>
</tr>
<tr class="odd">
<td>18</td>
<td align="right">8.6</td>
<td align="right">7</td>
</tr>
<tr class="even">
<td>19</td>
<td align="right">8.6</td>
<td align="right">7</td>
</tr>
<tr class="odd">
<td>20</td>
<td align="right">8.6</td>
<td align="right">7</td>
</tr>
<tr class="even">
<td>21</td>
<td align="right">8.6</td>
<td align="right">7</td>
</tr>
</tbody>
</table>
<hr />
<h2 id="experimental-data-based-classification-and-ranking-of-top-candidate-features-based-on-the-exir-model">Experimental-data-based classification and ranking of top candidate features based on the <code>ExIR</code> model</h2>
<p><strong>ExIR</strong> : <code>ExIR</code> is a model for the classification and ranking of top candidate features. The input data could come from any type of experiment such as transcriptomics and proteomics. This model is based on multi-level filteration and scoring based on several supervised and unsupervised analyses followed by the classification and integrative ranking of top candidate features. Using this function and depending on the input data and specified arguments, the user can get 1 to four tables.</p>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb21-1" data-line-number="1"></a>
<a class="sourceLine" id="cb21-2" data-line-number="2"><span class="co">#Preparing the required arguments for the `exir` function</span></a>
<a class="sourceLine" id="cb21-3" data-line-number="3">MyDesired_list &lt;-<span class="st"> </span>Desiredlist</a>
<a class="sourceLine" id="cb21-4" data-line-number="4">MyDiff_data &lt;-<span class="st"> </span>Diffdata</a>
<a class="sourceLine" id="cb21-5" data-line-number="5">Diff_value &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="dv">1</span>,<span class="dv">3</span>,<span class="dv">5</span>)</a>
<a class="sourceLine" id="cb21-6" data-line-number="6">Regr_value &lt;-<span class="st"> </span><span class="dv">7</span></a>
<a class="sourceLine" id="cb21-7" data-line-number="7">Sig_value &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="dv">2</span>,<span class="dv">4</span>,<span class="dv">6</span>,<span class="dv">8</span>)</a>
<a class="sourceLine" id="cb21-8" data-line-number="8">MyExptl_data &lt;-<span class="st"> </span>Exptldata</a>
<a class="sourceLine" id="cb21-9" data-line-number="9">Condition_colname &lt;-<span class="st"> &quot;condition&quot;</span></a>
<a class="sourceLine" id="cb21-10" data-line-number="10"></a>
<a class="sourceLine" id="cb21-11" data-line-number="11"><span class="co">#Running the ExIR model</span></a>
<a class="sourceLine" id="cb21-12" data-line-number="12">My.exir &lt;-<span class="st"> </span><span class="kw">exir</span>(<span class="dt">Desired_list =</span> MyDesired_list,</a>
<a class="sourceLine" id="cb21-13" data-line-number="13"><span class="dt">Diff_data =</span> MyDiff_data, <span class="dt">Diff_value =</span> Diff_value,</a>
<a class="sourceLine" id="cb21-14" data-line-number="14"><span class="dt">Regr_value =</span> Regr_value, <span class="dt">Sig_value =</span> Sig_value,</a>
<a class="sourceLine" id="cb21-15" data-line-number="15"><span class="dt">Exptl_data =</span> MyExptl_data, <span class="dt">Condition_colname =</span> Condition_colname,</a>
<a class="sourceLine" id="cb21-16" data-line-number="16"><span class="dt">verbose =</span> <span class="ot">FALSE</span>)</a>
<a class="sourceLine" id="cb21-17" data-line-number="17"></a>
<a class="sourceLine" id="cb21-18" data-line-number="18"><span class="kw">names</span>(My.exir)</a>
<a class="sourceLine" id="cb21-19" data-line-number="19"><span class="co">#&gt; [1] &quot;Driver table&quot;          &quot;DE-mediator table&quot;     &quot;nonDE-mediators table&quot;</span></a>
<a class="sourceLine" id="cb21-20" data-line-number="20"><span class="co">#&gt; [4] &quot;Biomarker table&quot;</span></a></code></pre></div>

</body>
</html>
